packages/lesswrong/server/editor/utils.ts

```diff
@@ -16,6 +16,10 @@ Utils.trimEmptyLatexParagraphs = (dom) => {
           len--;
       }
   }
+  const [firstLatexElement] = dom.getElementsByClassName("mjx-chtml");
+  const styleNode = dom.createElement("style");
+  styleNode.innerHTML = css;
```

Comment:
Change this to `innerText`. I don't see a specific vulnerability here, but it wouldn't be super surprising if there were a LaTeX program that would control part of the stylesheet, which would be XSS.

Reply:
Alas, when I replaced it with innerText it broke and stopped working. Not sure why.

Reply:
`textContent` worked though